src/Meilisearch/Index.cs

```diff
@@ -39,6 +39,18 @@ public Index(string uid, string primaryKey = default)
         /// </summary>
         public string PrimaryKey { get; internal set; }
 
+        /// <summary>
+        /// Fetch the primary key of the index.
+        /// </summary>
+        /// <returns>Primary key of the index fetch.</returns>
+        public async Task<string> FetchPrimaryKey()
+        {
+            var response = await this.client.GetAsync($"indexes/{this.Uid}");
```

Comment:
use `getIndex`

Reply:
I added the `FetchInfo()` method like in others SDKs and use it in the `GetIndex()` method too.